src/injections/pollingAction/index.js

```diff
@@ -0,0 +1,46 @@
+function pollingAction(action) {
+  const {
+    type,
+    target,
+    service,
+    payload = () => {},
+    successSelector = response => response.data,
+    failureSelector = response => response.problem,
+    timeout = 5000,
+    shouldRetry,
+    determination = response => response.ok
+  } = action;
+  const selector = typeof payload === 'function' ? payload : () => payload;
+
+  return {
+    prebehavior: dispatch => dispatch({ type, target }),
+    apiCall: async getState => service(selector(getState())),
+    determination,
+    success: (dispatch, response) =>
+      dispatch({
+        type: `${type}_SUCCESS`,
+        target,
+        payload: successSelector(response),
+        isPolling: true
+      }),
+    failure: (dispatch, response, state) => {
```

Comment:
Add `return true`, otherwise `withPostFailure` will not be executed

Reply:
Are you sure? We currently have
```
function composeInjections(...injections) {
  const injectionsDescription = mergeInjections(injections);

  const {
    prebehavior = () => {},
    apiCall = () => {},
    determination = () => true,
    success = () => true,
    postSuccess = () => {},
    postBehavior = () => {},
    postFailure = () => {},
    failure = () => {},
    statusHandler = () => true
  } = injectionsDescription;

  return async (dispatch, getState) => {
    prebehavior(dispatch);
    const response = await apiCall(getState);
    postBehavior(dispatch, response);
    if (determination(response)) {
      const shouldContinue = success(dispatch, response, getState());
      if (shouldContinue) postSuccess(dispatch, response, getState());
    } else {
      const shouldContinue = statusHandler(dispatch, response, getState());
      if (shouldContinue) {
        failure(dispatch, response, getState());
        postFailure(dispatch, response, getState());
      }
    }
  };
}
```

Both `failure` and `postFailure` depend on `statusHandler` and we don't use `statusHandler` here.